types/react-i18next/index.d.ts

```diff
@@ -1,24 +1,40 @@
-// Type definitions for react-i18next 4.6
```

Comment:
Could you back up the old version to a folder called v4 as described in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)?

Reply:
Old version has been backed up in v4 Folder